atomic-structure/content.md

```diff
@@ -0,0 +1,25 @@
+# Radioactive Decay
+#### Learning objective
+Basic understanding of atomic structure
```

Comment:
This should probably be in the guide. I'm fine with however the learning objectives should be constructed. I'm currently working on only really basic material so it's hard to think of clear objectives.